components/raftstore-v2/src/operation/ready/mod.rs

```diff
@@ -329,7 +345,7 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         let ready_number = ready.number();
         let mut write_task = WriteTask::new(self.region_id(), self.peer_id(), ready_number);
         self.storage_mut()
-            .handle_raft_ready(&mut ready, &mut write_task);
+            .handle_raft_ready(&mut ready, &mut write_task, ctx);
```

Comment:
order better be: (ctx, ready, write_task)

Reply:
fixed